src/main/kotlin/repository/AssignmentRepository.kt

```diff
@@ -14,4 +15,5 @@ interface AssignmentRepository : PagingAndSortingRepository<Assignment, Long> {
     fun findAllByPersonUuid(personUuid: UUID, page: Pageable): Page<Assignment>
     fun findAllByPersonUserCode(userCode: String, page: Pageable): Page<Assignment>
     fun deleteByCode(code: String)
+    fun findAllByToAfterOrToNull(to: LocalDate?, page: Pageable): Page<Assignment>
```

Comment:
To me it made sense to be able to look for Assignments without a 'To' date by looking for an assignment with to = null, however it can indeed not be null (throws a java.lang.IllegalArgumentException)